annual-calendar-creator/editorextensions/annual-calendar-creator/src/calendarsettingsmodal.ts

```diff
@@ -0,0 +1,30 @@
+import { EditorClient, Modal } from 'lucid-extension-sdk';
+import { createAnnualCalendar } from './extension';
+
+export interface CalendarSettingsModalMessage {
+    'year': number;
+    'language': string;
+    'localizedMonths': string[],
+    'localizedDays': string[]
```

Comment:
Some of the formatting here looks strange with inconsistencies between using ; and ,

Reply:
Good catch